web_src/js/features/formatting.js

```diff
@@ -13,9 +16,28 @@ export function initFormattingReplacements() {
     }
   }
 
-  // for each <time></time> tag, if it has the data-format="date" attribute, format
-  // the text according to the user's chosen locale
-  for (const timeElement of document.querySelectorAll('time[data-format="date"]')) {
-    timeElement.textContent = dateFormatter.format(new Date(timeElement.dateTime));
+  // for each <time></time> tag, if it has the data-forma attribute, format
+  // the text according to the user's chosen locale and formatter
+  formatAllTimeElements();
+}
+
+function formatAllTimeElements() {
+  const allTimeElements = document.querySelectorAll('time[data-format]');
+  for (const timeElement of allTimeElements) {
```

Comment:
`all` is a repeating if you have Element**s** already in the variable name.
```suggestion
  const timeElements = document.querySelectorAll('time[data-format]');
  for (const timeElement of timeElements) {
```